CryptoTechReminderSystem.Test/Gateway/HarvestGatewayTests.cs

```diff
@@ -39,7 +39,7 @@ public void TearDown()
             private void SetUpUsersApiEndpoint(string id, string firstName, string lastName, string email)
             {
                 var json = $"{{  \"users\":[    {{      \"id\":{id},      \"first_name\":\"{firstName}\"" +
-                           $",      \"last_name\":\"{lastName}\",       \"email\":\"{email}\" , \"is_active\":\"true\"   }}  ]}}";
+                           $",      \"last_name\":\"{lastName}\",       \"email\":\"{email}\" , \"is_active\":\"true\", \"roles\": [\"Delivery Principal\"]  }}  ]}}";
```

Comment:
This my be a cleaner way to set up api response
`new { last_name = lastName, email = email, is_active = true, etc... }`

Reply:
Will be sorted in a separate PR